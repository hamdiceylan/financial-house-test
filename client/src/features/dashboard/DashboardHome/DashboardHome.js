import React from 'react'
import { Grid } from 'semantic-ui-react'
import { Switch, Route, Redirect } from 'react-router-dom'
import DashboardNav from '../DashboardNav/DashboardNav'
import ClientDetail from '../ClientDetail/ClientDetail';
import TransactionList from '../TransactionList/TransactionList';
import TransactionDetail from '../TransactionDetail/TransactionDetail';


const DashboardHome = () => {
  return (
    <React.Fragment>
      <Grid stackable centered>
        <Grid.Column width={16}>
          <DashboardNav/>
        </Grid.Column>
      </Grid>
      <Switch>
        <Redirect exact from='/dashboard' to='dashboard/home'/>
        <Route path='/dashboard/home' render={() => <TransactionList />}/>
        <Route path='/dashboard/client-detail/:id' component={ClientDetail}/>
        <Route path='/dashboard/transaction-detail/:id' component={TransactionDetail}/>
        <Route path='/dashboard/transactions' render={() => <TransactionList />}/>
      </Switch>
    </React.Fragment>
  )
}

export default DashboardHome
