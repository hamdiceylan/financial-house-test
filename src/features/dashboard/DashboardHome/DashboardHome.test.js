import React from 'react'
import { shallow } from 'enzyme'
import DashboardHome from './DashboardHome';


describe('DashboardHome', () =>{
    const dashboardHome = shallow(<DashboardHome />);

    it('renders correctly', () => {
        expect(dashboardHome).toMatchSnapshot();
    })

})

