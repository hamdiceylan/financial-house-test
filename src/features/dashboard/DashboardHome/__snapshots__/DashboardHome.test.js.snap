// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardHome renders correctly 1`] = `ShallowWrapper {}`;
